src/sdk/pynni/nni/compression/torch/builtin_pruners.py

```diff
@@ -5,7 +5,7 @@
 import torch
 from .compressor import Pruner
 
-__all__ = ['LevelPruner', 'AGP_Pruner', 'FPGMPruner', 'L1FilterPruner', 'SlimPruner']
```

Comment:
Since L1 pruner and FPGM pruner are merged into rank pruner.
Documents and examples need to be updated as well.

Reply:
Thanks for review, I'll update them later